lib/ts/nextjs/ssr.ts

```diff
@@ -211,18 +201,22 @@ export const init = SuperTokensNextjsSSRAPIWrapper.init;
 export const getServerComponentSession = SuperTokensNextjsSSRAPIWrapper.getServerComponentSession;
 export const getServerActionSession = SuperTokensNextjsSSRAPIWrapper.getServerActionSession;
 export const getServerSidePropsSession = SuperTokensNextjsSSRAPIWrapper.getServerSidePropsSession;
-export const authenticateServerAction = SuperTokensNextjsSSRAPIWrapper.authenticateServerAction;
-export type AuthenticatedServerAction<T extends (...args: any[]) => any> = T extends (...args: infer A) => infer R
-    ? (session?: LoadedSessionContext, ...originalArgs: A) => R
-    : never;
+export const confirmAuthenticationAndCallServerAction =
+    SuperTokensNextjsSSRAPIWrapper.confirmAuthenticationAndCallServerAction;
 
 function getAuthPagePath(redirectPath: string): string {
     const authPagePath = SuperTokensNextjsSSRAPIWrapper.getConfigOrThrow().appInfo.websiteBasePath || "/auth";
     return `${authPagePath}?${FORCE_LOGOUT_PATH_PARAM_NAME}=true&${REDIRECT_PATH_PARAM_NAME}=${redirectPath}`;
 }
 
+function getRefreshAPIPath(): string {
+    const apiPath = SuperTokensNextjsSSRAPIWrapper.getConfigOrThrow().appInfo.apiBasePath || DEFAULT_API_PATH;
+    return `${apiPath}/session/refresh`;
+}
+
 function getRefreshLocation(redirectPath: string): string {
-    return `${SESSION_REFRESH_API_PATH}?${REDIRECT_PATH_PARAM_NAME}=${redirectPath}`;
+    const refreshAPIPath = getRefreshAPIPath();
+    return `${refreshAPIPath}?${REDIRECT_PATH_PARAM_NAME}=${redirectPath}`;
 }
```

Comment:
I think the first name is also present in the middleware file, you could indicate in the name that one path is meant for the frontend/as a full page redirect.